server/app/lib/actions/fusor/deployment/open_shift/install_image.rb

```diff
@@ -0,0 +1,102 @@
+#
+# Copyright 2015 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public
+# License as published by the Free Software Foundation; either version
+# 2 of the License (GPLv2) or (at your option) any later version.
+# There is NO WARRANTY for this software, express or implied,
+# including the implied warranties of MERCHANTABILITY,
+# NON-INFRINGEMENT, or FITNESS FOR A PARTICULAR PURPOSE. You should
+# have received a copy of GPLv2 along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+require 'net/scp'
+
+module Actions
+  module Fusor
+    module Deployment
+      module OpenShift
+        class InstallImage < Actions::Fusor::FusorBaseAction
+          def humanized_name
+            _("Transfer RHEL Guest 7.2 image to Virtualization Environment")
+          end
+
+          def plan(deployment, image_file_name)
+            super(deployment)
+            plan_self(deployment_id:   deployment.id,
+                      image_file_name: image_file_name)
+          end
+
+          def run
+            ::Fusor.log.debug "================ InstallImage run method ===================="
+            begin
+
+              deployment = ::Fusor::Deployment.find(input[:deployment_id])
+
+              from_image_path    = "/usr/share/#{input[:image_file_name]}/#{input[:image_file_name]}*/"
+              from_tar_file_path = "/tmp"
+              to_img_file_path   = "/root"
+              tar_file_name      = "#{input[:image_file_name]}.tar.gz"
+
+              tar_cmd = "cd  #{from_image_path} && "\
+                        "tar cfvz  /#{from_tar_file_path}/#{tar_file_name} ./* && "\
+                        "cd -"
+
```

Comment:
@jmrodri I'm tar'ing up the folder because, we need to keep the folder structure with all its contents.  Like @jwmatthews mentioned, we cannot simply just use a single file in this case